### DISCUSSION TITLE
Why queryset is needed in django modelselect2widget

### DISCUSSION THREAD

**Discussion by harishsoccer7** [codingjoe/django-select2#213] [Q&A]
Why do we need to mention queryset as parameter in django select2 widget since i am using view function and model object to render the response each time

**Comment by codingjoe** [comment]
Hi @harishsoccer7,

Thank you for reaching out. I am afraid that I don't yet fully understand your question. Would you be so kind and maybe share some of your code as an example?

Best
Joe
